crates/ibc/src/trace.rs

```diff
@@ -108,13 +108,24 @@ pub fn is_nft_trace(
     }
 }
 
-/// Return true if the source of the given IBC trace is this chain
-pub fn is_sender_chain_source(
+/// Returns true if the denomination originally came from the sender chain, and
+/// false otherwise.
+pub fn is_sender_chain_source_str(
```

Comment:
just curious, why did you add `_str`?

Reply:
I added `_str` in order to distinguish https://github.com/anoma/namada/blob/763b5fc0b251144271d6d6ccf134aab76c91eacb/crates/ibc/src/lib.rs#L825 from https://github.com/anoma/namada/blob/763b5fc0b251144271d6d6ccf134aab76c91eacb/crates/ibc/src/lib.rs#L524 since they are both used in the same file. That being said, I should have just used import `as`.

Reply:
I've now reverted the naming and use import `as`.